index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700&family=Roboto+Serif:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="head">
+      <h3>My Logo</h3>
```

Comment:
- [ ] Great job so far @cosywasswa your project looks great. There are just  a few adjustments you need to make and you'll be on your way to the next project :wink: 
- [ ] I will suggest that since this text is serving as your logo, It needs to be wrapped in a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :100: 
- [ ] Lastly, your design seems to have some white space on the top and left sides of the design, even though it looks small it doesn't give your design a pleasant look and feel and it's making it look slightly different from the Figma design. Apart from that, everything else looks great. So I will suggest you fix that as well  :wink: 

| Your Design | Figma Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/234847634-d3eda8b1-1246-4b7d-bf87-72c05f204df1.png) | ![image](https://user-images.githubusercontent.com/102699958/234847933-87c6f864-e521-4d1a-b24c-b84ccaf84e61.png) |